doc/nrf/libraries/networking/nrf_cloud_log.rst

```diff
@@ -0,0 +1,149 @@
+
+Each JSON log message contains the following elements:
+
+* appId - Set to "LOG".
+* ts - Timestamp if accurate time is known, in the Unix epoch.
+* seq - Sequence number to help order messages sent simultaneously, as well as to sort messages used when no timestamp is available.
+* dom - Logging domain ID; usually 0.
+* src - Name of the C function or subsystem which generated the log.
+* lvl - Log level. 0 = NONE (off), 1 = ERR, 2 = WRN, 3 = INF, 4 = DBG.
+* msg - The message string.
+
+The library sets the sequence number and timestamp automatically.
+
+Log messages displayed by a device through a UART or RTT connection show timestamps that reset to 0 when the device resets.
+Cloud-based log messages show timestamps with real date and time values for when each message was generated.
+Using real date and time for timestamps is important because log messages are stored by default for up to 30 days.
+
+The `nRF Cloud`_ website displays a device's log messages and enables the user to download historical log messages.
+Log messages can also be retrieved using `the ListMessages REST API <nRF Cloud REST API ListMessages_>`_.
+
+Implementation
+==============
+
+Supported features
+==================
+
+If the :ref:`lib_date_time` library is enabled, and the current date and time are known, the timestamp field of the alert will be automatically set accordingly.
+The sequence number will be set to a monotonically-increasing value, which resets to 0 when the device restarts.
+
+Supported backends
+==================
+
+When so configured, this library includes a Zephyr logging backend which can transport log messages to nRF Cloud using MQTT or REST.
+The logging backend can also use either JSON messages or dictionary-based compact binary messages.
+
+Multiple JSON log messages are sent together as a JSON array to the `d2c/bulk device message topic <nRF Cloud MQTT Topics_>`_.
+The nRF Cloud backend will split the array into individual JSON messages for display.
+Use of the bulk topic reduces data transfer size by consolidating MQTT or REST message overhead for multiple messages.
+
+Multiple dictionary-based log messages are sent together as a binary message to the `d2c/bin device message topic <nRF Cloud MQTT Topics_>`_.
+The nRF Cloud website will not display the contents of the dictionary-based log messages in real time.
+Instead, you must download a binary file containing the logs over a certain range of time, then decode the logs using a Python script and the dictionary built when the application was built.
+
+Requirements
+************
+
+The device must be connected to nRF Cloud before calling the :c:func:`nrf_cloud_log_send` function.
+The :c:func:`nrf_cloud_rest_log_send` function initiates the connection as needed.
+
+Configuration
+*************
+
+Configure one or both of the following options to enable direct log messages or the logging backend:
```

Comment:
I'm not sure I saw a statement which said what happens if you enable both direct log messages and the logging backend

Reply:
Actually found it in the header file. I'm thinking if we should mention it here.